week-2-practise/level-2/1-boolean.js

```diff
@@ -4,7 +4,7 @@
 
 var isHappy = "false";
 
-if (isHappy) {
+if (!isHappy) {
```

Comment:
Not quite - `isHappy` is a string, and usually you'd want a boolean in an `if` statement. How can yu modify the `isHappy` variable then?